Update typography guidance and restructure styles section (#3343)
alphagov/govuk-design-system: src/styles/paragraphs/index.md

---
title: Paragraphs
description: Use paragraphs for your core content

> section: Styles

Author: ```suggestion
section: Styles
backlogIssueId: 64
```

theme: Typography
layout: layout-pane.njk
showPageNav: true
order: 8
---

{% from "_example.njk" import example %}

> The default paragraph font size is 19px on large screens and 16px on small screens.

Author: ```suggestion
The default paragraph font size is 19px.
```

Updated to be in line with the new scale – should we say anything else here to say that this has recently changed and link to an explanation? This might be a pattern we repeat on other pages where the values have changed...

Reviewer: Agree. Is there anything we can take from the WCAG 2.2 work here?

Author: Maybes. Summat to chat about next week!

Reviewer 2: I'm not sure about WCAG 2.2. Would this work? [text spacing](https://www.w3.org/TR/WCAG22/#text-spacing)

Author: chat with Charlotte about using the WCAG thing
mock it up: one with wcag banner, one using just text

{{ example({ group: "styles", item: "paragraphs", example: "body", html: true, open: true }) }}

You can also add classes to create a lead paragraph or smaller body copy to convey hierarchy in your page.

## Lead paragraph

A lead paragraph is an introductory paragraph that you can use at the top of a page to summarise the content. Lead paragraphs use 24px type on desktop and should only be used once per page if needed.

{{ example({ group: "styles", item: "paragraphs", example: "lead", html: true, open: true }) }}

## Body small

> You can use the `govuk-body-s` class sparingly to make your paragraph font size smaller: 16px on larger screens and 14px on smaller screens.

Author: ```suggestion
You can use the `govuk-body-s` class sparingly to make your paragraph font size 16px instead of 19px.
```
